webpack.config.js

```diff
@@ -1,102 +1,92 @@
+}())
+
+const externalAssets = {
+  css: [
+    zendeskGardenJsDelivrUrl
+  ],
+  js: [
+    'https://assets.zendesk.com/apps/sdk/2.0/zaf_sdk.js'
+  ]
+}
 
 module.exports = {
   entry: {
     app: [
-      "babel-polyfill",
-      "./src/javascripts/index.js",
-      "./src/stylesheets/app.scss"
+      'babel-polyfill',
+      './src/javascripts/locations/ticket_sidebar.js',
+      './src/index.css'
     ]
   },
+  mode: process.env.NODE_ENV,
   output: {
-    path: path.resolve(__dirname, "./dist/assets"),
-    filename: "main.js",
-    sourceMapFilename: "[file].map"
+    filename: '[name].js',
+    path: path.resolve(__dirname, 'dist/assets')
   },
   module: {
     rules: [
       {
         test: /\.js$/,
-        exclude: /node_modules/,
-        enforce: "pre",
-        loader: "eslint-loader"
-      },
-      {
-        test: /\.(gif|jpe?g|png|svg|woff2?|ttf|eot)$/,
-        loader: "url-loader?limit=10000&name=[name].[ext]"
-      },
-      {
-        test: /\.scss$/,
-        use: extractStyles.extract({
-          fallback: "style-loader",
-          use: [
-            "css-loader?sourceMap&root=" +
-              path.resolve(__dirname, "./dist/assets"),
-            "sass-loader?sourceMap"
-          ]
-        })
+        use: { loader: 'babel-loader' }
       },
       {
+        type: 'javascript/auto',
         test: /\.json$/,
-        include: path.resolve(__dirname, "./src/translations"),
-        loader: "translations-loader",
-        options: {
-          runtime: "handlebars"
-        }
-      },
-      {
-        test: /\.js$/,
-        exclude: /node_modules/,
-        loader: "babel-loader"
+        include: path.resolve(__dirname, './src/translations'),
+        use: './webpack/translations-loader'
       },
       {
-        test: /\.(handlebars|hd?bs)$/,
-        loader: "handlebars-loader",
-        options: {
-          extensions: [".handlebars", ".hdbs", ".hbs"],
-          runtime: "handlebars",
-          inlineRequires: "/images/"
-        }
+        test: /\.(sa|sc|c)ss$/,
+        use: [
+          MiniCssExtractPlugin.loader,
+          {loader: 'css-loader', options: { url: false }},
+          'postcss-loader'
+        ]
       }
     ]
   },
-  resolveLoader: {
-    modules: ["node_modules", path.resolve(__dirname, "./lib/loaders")]
-  },
-  resolve: {
-    modules: ["node_modules", path.resolve(__dirname, "./lib/javascripts")],
-    alias: {
-      app_manifest: path.resolve(__dirname, "./dist/manifest.json")
-    }
-  },
-  externals: {
-    handlebars: "Handlebars",
-    jquery: "jQuery",
-    lodash: "_",
-    moment: "moment",
-    zendesk_app_framework_sdk: "ZAFClient"
-  },
-  devtool: "#eval",
+
   plugins: [
-    extractStyles,
+    // Empties the dist folder
+    new CleanWebpackPlugin(['dist/*']),
+
+    // Copy over static assets
+    new CopyWebpackPlugin([
+      { from: 'src/manifest.json', to: '../', flatten: true },
+      { from: 'src/images/*', to: '.', flatten: true }
+    ]),
```

Comment:
Hi Tony. Is it planned to also copy over the translation folder? As it stand right now running 'zat validate' against the 'dist' directory after bundling yields an error that 'translations/en.json' is missing:
> validate  Missing translation file for locale 'en'. Learn more: http://developer.zendesk.com/documentation/apps/i18n.html

Reply:
Ah, I see the translations-loader loader in webpack.config.js but it doesn't look like it's moving things over for me.

Reply:
Nice catch @bryan-flynn-zd! It should work now. `translations-plugin.js` was deleted by accident.
We rebuilt it to only extract info from en.json, because Zendesk Marketplace won't accept external translations.